lib/conjur/command/users.rb

```diff
@@ -136,7 +136,7 @@ class Conjur::Command::Users < Conjur::Command
       c.flag [:p,:password]
 
       c.action do |global_options,options,args|
-        username, password = Conjur::Authn.read_credentials
+        username, password = Conjur::Authn.get_credentials
```

Comment:
If you use `read_credentials`, you can only change a user's password if his credentials are stored in his netrc. Using `get_credentials`, you can update his password however his credentials are provided (e.g. through the environment).